content/docs/get-started-with-neon/using-api-keys.md

```diff
@@ -12,8 +12,77 @@ Once authenticated, a user can create and access Projects and [query Project dat
 
 ## Using API keys as a Neon User
 
-API keys allow users to access Neon application programming interface.
+API keys allow users to access the Neon application programming interface.
 
```

Comment:
I know you didn't add this @WeaponXII, but I find the whole section on the `An API key provides access...` confusing. The information is very contextual and may not be the first thing a user needs to know in order to get started. I do have some questions:

1. Is it mission critical for users to know that API keys cannot be scoped to specific Projects? If so this statement needs more details.

2. We should consider rewriting this section to improve readability. `Provision multiple API Keys` is an accurate statement, but it might be too jargony for a "Getting Started Section".